boefjes/boefjes/plugins/kat_ssl_scan/main.py

```diff
@@ -1,15 +1,44 @@
 import subprocess
 
+TLS_CAPABLE_SERVICES = ("https", "ftps", "smtp", "smtps", "imaps", "pop3s", "ssh", "rpd")
```

Comment:
SSH isn't TLS. RDP is called `ms-wbt-server` by nmap which is also the service name that we copy from nmap.

```suggestion
TLS_CAPABLE_SERVICES = ("https", "ftps", "smtp", "smtps", "imaps", "pop3s", "ms-wbt-server")
```